python/GafferImageTest/OpenImageIOReaderTest.py

```diff
@@ -515,6 +515,19 @@ def testOffsetAlignment( self ) :
 
 				self.assertImagesEqual( r["out"], offsetIn["out"], ignoreMetadata = True )
 
+	def testFileNameContext( self ) :
+
+		s = Gaffer.ScriptNode()
+		s["reader"] = GafferImage.OpenImageIOReader()
+
+		s["expression"] = Gaffer.Expression()
+		s["expression"].setExpression( 'parent["reader"]["fileName"] = "%s"' % self.fileName )
+
+		with Gaffer.ContextMonitor( root = s["expression"]["out"] ) as cm :
```

Comment:
Looks like we have a trivial CI error :

```
Error: ERROR: testFileNameContext (GafferImageTest.OpenImageIOReaderTest.OpenImageIOReaderTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/__w/gaffer/gaffer/build/python/GafferImageTest/OpenImageIOReaderTest.py", line 526, in testFileNameContext
    with Gaffer.ContextMonitor( root = s["expression"]["out"] ) as cm :
KeyError: "'out' is not a child of 'expression'"
```

Not sure how that could have passed at your end - perhaps you have a backwards compatibility shim for `out`? I think the simplest fix is just `root = s["expression"]`.